docs/setup-guide.md

```diff
@@ -0,0 +1,94 @@
+# Setup guide
+
+This documentation will walk you through setting up a portfolio website with CodersCard project.
+
+## Prerequisites
+
+- [GitHub account](https://github.com/join)
+- [Netlify account](https://app.netlify.com/signup) - you can login with GitHub
+
+## Setup
+
+1. Fork [CodersCard](https://github.com/CodersCrew/CodersCard) to your account.
+
+2. Go to [your Netlify account](https://app.netlify.com/) and click on the **New site from Git** button.
+
+3. Complete a quick 3-step process to setup your website.
```

Comment:
I see four elements/steps on the list

Reply:
Oh this 3-step process is part of their website, I will mention that